k8s/examples/wordpress/mysql/15-configmap.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: wordpress-mysql
+  namespace: default
+  labels:
+    app: wordpress
+    tier: mysql
+type: Opaque
+data:
+  mysql-root-password: "cGFzc3dvcmQ="
```

Comment:
Maybe you could add a comment that explains that this is a base64 encoded password so people can sub out their own?